File: 2015/rust/aoc/tests/test_day02.rs
use aoc::day02;

#[test]
fn test_part1() {
    assert_eq!(day02::part1("./data/day02/sample"), 58 + 43);
    assert_eq!(day02::part1("./data/day02/input"), 1598415);
}

#[test]
fn test_part2() {
    assert_eq!(day02::part2("./data/day02/sample"), 34 + 14);
    assert_eq!(day02::part2("./data/day02/input"), 3812909);
}

File: 2019/python/tests/test_day02.py
from pathlib import Path

from aoc2019 import day02

DATA_DIR = Path(__file__).parent.parent / "data/day02"


def test_part1():
    assert day02.part1(DATA_DIR / "input.txt") == 4930687


def test_part2():
    assert day02.part2(DATA_DIR / "input.txt") == 5335
